app/assets/javascripts/Components/autotest_manager.jsx

```diff
@@ -316,6 +317,16 @@ class AutotestManager extends React.Component {
             disableActions={{rename: true}}
           />
         </fieldset>
+        <fieldset>
+          <legend><span>{I18n.t("automated_tests.solution_files")}</span></legend>
+          <SubmissionFileManager
+            readOnly={!this.state.enable_test}
+            assignment_id={this.props.assignment_id}
+            grouping_id={this.props.grouping_id}
+            enableSubdirs={true}
+            is_solution_file={true}
```

Comment:
If the only purpose of this prop is to decide whether or not to render the file viewer, rename the prop to something that describes its function